### ISSUE TITLE
QUESTION: Dofus Unity support?

### ISSUE THREAD

**Issue by brunoorsolon** [Antiz96/Ankama-Launcher-Container#62]
Hello

Are there plans to support Dofus Unity?

**Comment by Antiz96** [comment]
Hi,

**TL;DR:**

Yes, it is most likely already supported and playable right now actually (via the "Dofus Beta" game in the launcher). But chances are that the `Ankama-Launcher-Container` won't be required anymore to play Dofus Unity once it's officially released (see below for more details).

**Detailed answer:**

As long as it is integrated into the Ankama Launcher, Dofus Unity should work. Actually, it is most likely playable right now via the "Dofus Beta" game from the launcher. I haven't tried it myself yet, but I've seen people playing Dofus Unity on Linux so no reason it doesn't already work with the `Ankama-Launcher-Container`.

However, with the Unity version, Ankama is eventually aiming at having it running natively on Linux.  
Quoted from [this](https://www.dofus.com/fr/mmorpg/actualites/news/1672928-dofus-unity-resume-live-20-12-2023) page (translated from french):

> In terms of compatibility, the Unity version of DOFUS will be compatible on Windows and MacOS for the time being and, as one of our developers is hard at work on it, we also have a native Linux version that lets you run DOFUS under Unity without going through Wine.

There are also rumors that Ankama is looking at (re-)introducing Dofus (Unity only?) into Steam, eventually allowing people to play it via Steam's proton (if required).

So yes, Dofus Unity *should* be supported and playable already (via the "Dofus Beta" game in the launcher) with the `Ankama-Launcher-Container`.  
However, if Ankama eventually distributes a Linux native version of Dofus Unity and / or introduce it to Steam, chances are that this `Ankama-Launcher-Container` project will not be required anymore *(at least for Dofus and Wakfu, which is already on Steam. I don't know for Dofus Retro though)* :)
